### PR TITLE
Liftoff: Add site support and fill seat

### PR DESCRIPTION
**Pull request by Vungle-GordonTian**
- Add site support for bid request, and transfer it into app request
- Fill seat in bid response

### CODE REVIEW

**Comment by Vungle-GordonTian** [comment]
@Sonali-More-Xandr @onkarvhanumante PTAL at this PR.

**Review by onkarvhanumante — DISMISSED** [review]

**Review by gargcreation1992 — COMMENTED** [review]
Thank you for being patient with us. Left one refactoring review that might help in improving code readibility.

File: adapters/liftoff/liftoff.go [hunk 1]
```diff
@@ -76,10 +76,24 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
...
+			if request.Site != nil {
+				requestCopy.Site = nil
+				requestAppCopy = openrtb2.App{
+					ID: bidderImpExt.PubAppStoreID,
+				}
+			} else {
+				errs = append(errs, errors.New("failed constructing app, must have app or site object in bid request"))
+				continue
+			}
+		}
```
[the remark this answers is not in the document]
Comment (Vungle-GordonTian) [reply adapters/liftoff/liftoff.go]: Done refactoring. PTAL.

**Review by gargcreation1992 — APPROVED** [review]

**Comment by Vungle-GordonTian** [comment]
@onkarvhanumante PTAL at the latest commit.

**Pull request merged**